common/ideas/02_zael_national_ideas.txt

```diff
@@ -0,0 +1,213 @@
+tunis_ideas = {
+	start = {
+		galley_power = 0.25
+		may_perform_slave_raid = yes
+	}
+	
+	bonus = {
+		naval_forcelimit_modifier = 0.3
+	}
+	
+	trigger = {
+		tag = TUN
+	}
+	
+	catalan_guard = {
+		infantry_power = 0.15
+	}
+	dual_diplomacy = {
+		rival_change_cost = -1
+		diplomatic_upkeep = 1
+	}
+	corsairs = {
+		navy_tradition = 1
+		privateer_efficiency = 0.25
+	}
+	attract_foreign_pirates = {
+		global_sailors_modifier = 0.25
+		regiment_manpower_usage = -0.15
+	}
+	board_of_captains = {
+		leader_navy_fire = 1
+		leader_army_fire = 1
+	}
+	export_monopolies = {
+		trade_efficiency = 0.15
+		embargo_efficiency = 0.25
+	}
+	tunisian_caliphate = {
+		own_tolerance = 2
+		monthly_piety_accelerator = 0.001
+	}
+}
+
+scotland_ideas = {
+	start = {
+		land_forcelimit_modifier = 0.33
+		leader_land_shock = 1
+	}
+	
+	bonus = {
+		land_morale = 0.15
+	}
+	
+	trigger = {
+		tag = SCO
+	}
+	
+	break_the_black_douglas = {
+		global_unrest = -2
+		tax_efficiency = 0.15
+	}
+	modernize_the_royal_scots_navy = {
+		global_ship_cost = -0.1
+	}
+	rein_in_the_clans = {
+		global_manpower_modifier = 0.2
+	}
+	comets = {
+		stability_cost_modifier = -0.1
+	}
+	basilikon_doron = {
+		legitimacy = 1
+		devotion = 1
+		republican_tradition = 0.3
+	}
+	pass_the_act_of_revocation = {
+		ae_impact = -0.15
+	}
+	remember_the_bruce = { # get stuck in ya blaggards
+		infantry_shock = 0.5
+	}
+}
+
+pomerania_ideas = { # feels weird to not type "pomeranian"
+	start = {
+		land_morale = 0.1
+		light_ship_cost = -0.2
+	}
+	
+	bonus = {
+		discipline = 0.075
+	}
+	
+	trigger = {
+		tag = POM
+	}
+	
+	legacy_of_pirates = {
+		embargo_efficiency = 0.33
+		privateer_efficiency = 0.33
+	}
+	pomeranian_merchants = {
+		merchants = 1
+		trade_steering = 0.25
+	}
+	nortern_ports = {
+		global_trade_power = 0.1
+	}
+	religious_freedom = {
+		heretic_tolerance = 3
+		max_heretic_tolerance = 2
+	}
+	mercenary_contracts = {
+		mercenary_maintenance_modifier = -0.2
+		spy_offence = 0.5
+	}
+	dominate_the_baltic = {
+		trade_efficiency = 0.25
+	}
+	germany_navy = {
+		naval_forcelimit_modifier = 0.25
+	}
+}
+
+arabia_ideas = {
+	start = {
+		movement_speed = 0.1
+		land_attrition = -0.33
+	}
+	
+	bonus = {
+		trade_efficiency = 0.2
+	}
+	
+	trigger = {
+		tag = ARB
+	}
+	
+	indian_ocean_trade = {
+		trade_steering = 0.5
```

Comment:
50% is a LOT. Rethink maybe?

Reply:
It may be too much yes (i'm not really sure how powerful trade steering passive pull is)
Justification for this is two part however:

1) In order to form this tag they'd need to have 5 of the most contested nodes. This is also an end-game tag. By the time arabia is formed they'll have someone like a Spain or a Hindustan gunning for them in a very active manner, and will have had to potentially fight ottomans, mamluks, persia, and/or ethiopia. Spain, meanwhile, may have had to fight a morrocco and/or france, and otherwise would be blitzing to egypt. Arabia is shit land and most it's nodes can be stolen from, so this eco boost is a big crutch in their chances of survival imo, and a good reward for the difficulty of going for the tag.

2) other than the mega morale (oh hey look, it's right down below this comment) arabia's only mil buff is that it will have the potential to field entirely cav based armies. unlike hordes, their cav army doesn't have terrain bonuses, and nowhere does arabia have CCA. Overall arabia is going to be have to rely on super economy to sustain their quality-less but unbreakable armies. but to be frank 25% isn't much more than france or spain have, especially if you consider the age bonuses and other national ideas those tags get.

My big concern is early game all the lilboi tags may get this idea set, in which case that steering is def a lot... but i think it's worth testing since it's "only" ~2x compared to normal.

Reply:
I guess... I just see it as natural that Hindustan, an entire subcontinent and possibly 1/3 of contemporary population, is more powerful than Arabia.

Reply:
Hindustan should still have more trade power due to sheer number of provinces, and the amount of boats that come with a large coastline